notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb

```diff
@@ -936,9 +1060,9 @@
      "output_type": "stream",
      "text": [
       "Benchmark FP32 model (IR)\n",
-      "[ INFO ] Throughput:   3610.98 FPS\n",
+      "[ INFO ] Throughput:   3755.92 FPS\n",
       "Benchmark INT8 model (IR)\n",
-      "[ INFO ] Throughput:   15613.06 FPS\n"
```

Comment:
I could not reproduce `15613.06` FPS on main branch, there is almost no difference between FPS before and after my changes, perhaps it is because of openvino performance degradation